engine.go

```diff
@@ -135,6 +138,10 @@ func WithOpenAPIConfig(config OpenAPIConfig) func(*Engine) {
 		}
 
 		WithMiddlewareConfig(config.MiddlewareConfig)(e)
+
+		if config.OpenApiGeneratorOptions != nil && len(config.OpenApiGeneratorOptions) > 0 {
+			e.OpenAPI.generator = openapi3gen.NewGenerator(config.OpenApiGeneratorOptions...)
+		}
```

Comment:
This overwrites the OpenAPI generator, which is created with default values from `NewServer()` -> `NewOpenApi()`.